source/php/Module/SubmitForm/views/fields/event_tags.blade.php

```diff
@@ -22,7 +21,6 @@
         'options' => [
             'loading'   => __('Loading', 'event-integration') . "...",
         ]
-    ])
+    ])4
```

Comment:
Hmm this view is used by the old form, 'typo commit'? :)

Reply:
woops